openedx/features/job_board/views.py

```diff
@@ -1,9 +1,21 @@
+from django.views.generic.list import ListView
 from edxmako.shortcuts import render_to_response
 
+from .models import Job
 
-def list_jobs(request):
 
-    return render_to_response('features/job_board/job_list.html', {})
+class JobsListView(ListView):
+
+    model = Job
+    context_object_name = 'jobs_list'
+    paginate_by = 10
+    template_name = 'features/job_board/job_list.html'
+    ordering = ['-created']
+
+    def get_context_data(self, **kwargs):
+        context = super(JobsListView, self).get_context_data(**kwargs)
+        context['total_job_count'] = Job.objects.all().count()
```

Comment:
we already have job_list in the context from above statement, do we need `Job.objects.all()` to run to get count?

Reply:
Because we are using pagination with `paginate_by = 10`, we only have 10 or less than 10 objects being passed to us in the context each time. Hence, the passing of `total_job_count`. Technically it is more efficient too since we are not grabbing all the records, but only 10 or less than 10 and then just calculating the total number of objects.

Reply:
We do need the total number of jobs in the database. When we implement the search feature, we will be updating this to show all the jobs that match the search filter. We cannot use `jobs_list.count` since it is 10 or less than 10 because of pagination, as I mentioned earlier.

Reply:
still you don't need this method to show total number of results, paginator will do this for you. 
Please remove this method and use `page_obj.paginator.count` on template side for total number of results. 

**Reference:**
https://docs.djangoproject.com/en/dev/ref/paginator/#attributes

Reply:
Done